pkg/porter/cnab.go

```diff
@@ -260,3 +268,14 @@ func (o *sharedOptions) combineParameters() map[string]string {
 
 	return final
 }
+
+// Validate that the provided driver is supported
+func (o *sharedOptions) validateDriver() error {
+	// Not using duffle's driver.Lookup() as it currently does not return an error on invalid drivers
```

Comment:
Is there an issue in duffle that this function needs to be changed? If not, can you make one, even just a placeholder, so that we don't forget?

Reply:
Good call; filed https://github.com/deislabs/duffle/pull/723.  We'll see if this gains approval...